tools/report-converter/codechecker_report_converter/coccinelle/output_parser.py

```diff
@@ -23,6 +23,11 @@ def __init__(self, analyzer_result):
         super(CoccinelleParser, self).__init__()
 
         self.analyzer_result = analyzer_result
+        self.checker_name = None
+
+        self.checker_name_re = re.compile(
```

Comment:
Unfortunatelly this regex will not work if the file name contains white spaces:
E.g.:
```
Processing my sample.cocci
```
Here the file name is `my simple.cocci`. I think it's very rare use case but maybe we can use the following regex which solve this problem too: `^Processing (?P<checker_name>[\S ]+)\.cocci$`.
Also after the `Processing` keyword we should expect only one white space and the `.cocci` extension should be at the end of the line (so no extra spaces can be found after it). What do you think @jay24rajput ?

Reply:
> Here the file name is `my simple.cocci`. I think it's very rare use case but maybe we can use the following regex which solve this problem too: `^Processing (?P<checker_name>[\S ]+)\.cocci$`.

Yes that makes sense, I will modify the regex to match the one stated by you!


> Also after the `Processing` keyword we should expect only one white space and the `.cocci` extension should be at the end of the line (so no extra spaces can be found after it). What do you think @jay24rajput ?

Well yes that is true, I just kept it that in case something messes up, the regex should not miss the line. Nevertheless, If you say I can remove those extra white spaces check before and after the `.cocci` file